pyro/infer/elbo.py

```diff
@@ -86,6 +133,15 @@ def __init__(
         self.jit_options = jit_options
         self.tail_adaptive_beta = tail_adaptive_beta
 
+    def __call__(
+        self, model: torch.nn.Module, guide: torch.nn.Module
+    ) -> torch.nn.Module:
```

Comment:
nit: refine the return type to `ELBOModule`?

Reply:
Done